Gemfile

```diff
@@ -6,7 +6,7 @@ source 'https://rubygems.org' do
   gem 'appraisal', '>= 2.4', '< 3'
   gem 'bundler', '>= 2.2', '< 3'
   gem 'mocha', '>= 2', '< 3'
-  gem 'rake', '>= 13', '< 14'
+  gem 'rake', '>= 13', '< 14', require: 'rake/testtask'
```

Comment:
Curious about this `require` versus `require` inside of `Rakefile`?

Reply:
It's optional, but brings our `Rakefile` more inline with a standard Rakefile that let's bundler handle requiring/loading files without the explicit need to.